src/Components/Common/ProfileDropdown.js

```diff
@@ -90,27 +90,14 @@ const ProfileDropdown = () => {
               <span className="align-middle">Taskboard</span>
             </Link>
           </DropdownItem>
-          <DropdownItem className="p-0">
-            <Link
-              to={process.env.PUBLIC_URL + "/pages-faqs"}
-              className="dropdown-item"
-            >
-              <i className="mdi mdi-lifebuoy text-muted fs-16 align-middle me-1"></i>{" "}
-              <span className="align-middle">Help</span>
-            </Link>
-          </DropdownItem>
           <div className="dropdown-divider"></div>
-          <DropdownItem className="p-0">
-            <Link
-              to={process.env.PUBLIC_URL + "/pages-profile"}
-              className="dropdown-item"
-            >
-              <i className="mdi mdi-wallet text-muted fs-16 align-middle me-1"></i>{" "}
-              <span className="align-middle">
-                Balance : <b>$5971.67</b>
-              </span>
-            </Link>
-          </DropdownItem>
+          {/* <DropdownItem className='p-0'>
```

Comment:
Please clean up the commented code.

Reply:
This commented code can be used in future, that's why I haven't deleted yet.

Reply:
For anything code that is to be deployed to staging and prod, no unnecessary comments are required. You can have them in your feature branch but we'll need them cleaned before we can merge them.